system/integrations/github.rst

```diff
@@ -0,0 +1,57 @@
+GitHub
+======
+
+Use GitHub integration to manage your GitHub issues right from within Zammad.
+
+.. figure:: /images/system/integrations/github/github-integration-page.png
+   :alt: Integration page for GitHub
+   :align: center
+   :width: 90%
+
+.. note:: GitHub integration does not support pull requests.
+
+Setup
+-----
+
+1. In your GitHub settings, create a new API token under
+   **Developer settings > Personal access tokens > Generate new token**.
+
+   Under **Select scopes**, choose ``public_repo`` only.
+   (If you wish to manage issues on a private repository,
+   select the parent scope ``repo`` instead.)
+
+   .. figure:: /images/system/integrations/github/add-api-token-github.gif
+      :alt: Screencast showing how to create a new API token
+      :align: center
+      :width: 90%
+
+      Create a new API key with the appropriate privileges.
+
+   .. hint:: Zammad can only use **one GitHub API token** at a time.
+      Make sure the GitHub account it belongs to has the privileges 
+      to manage (close/reopen) issues on any repos you’d like to track.
+
+2. Enter your new API token in Zammad and enable GitHub integration.
+
+   .. figure:: /images/system/integrations/github/configure-and-active-github-integration.gif
+      :alt: Screencast showing how to configure Zammad’s GitHub integration
+      :align: center
+      :width: 90%
+
+   .. hint:: Leave the default API endpoint (``https://api.github.com/graphql``) as-is
+      unless you’re using GitHub Enterprise Server.
+
+Once completed, a new tab for GitHub issues will appear in the ticket settings. 🎉
```

Comment:
I'm honestly not sure if I interpreted this part correctly. It might even help to add a screenshot here!

Do you mean to say that in the "New Ticket" dialog, there's now an option to make the ticket a GitHub issue?